frameworks/Java/mangooio/src/main/resources/ehcache.xml

```diff
@@ -0,0 +1,13 @@
+<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../config/ehcache.xsd" updateCheck="false">
```

Comment:
No, a Cache is not used at all. But the Framework has an configuration for ehCache which creates an instance of a Cache provider. I could remove the file, but ehCache would still autoconfigure.